experimental/segmenter/build.rs

```diff
@@ -86,7 +89,9 @@ fn set_break_state(
     right_index: usize,
     break_state: i8,
 ) {
-    if break_state_table[left_index * property_length + right_index] == UNKNOWN_RULE {
+    if break_state_table[left_index * property_length + right_index] == UNKNOWN_RULE
+        || break_state_table[left_index * property_length + right_index] == NOT_MATCH_RULE
+    {
         break_state_table[left_index * property_length + right_index] = break_state;
     }
 }
```

Comment:
Nit: Maybe use a variable to store `left_index * property_length + right_index`?